docs/buildah-build.1.md

```diff
@@ -407,6 +407,19 @@ By default, Buildah manages _/etc/hosts_, adding the container's own IP address.
 
 Set the OS of the image to be built, and that of the base image to be pulled, if the build uses one, instead of using the current operating system of the host.
 
+**--output**, **-o**=""
+
+Output destination (format: type=local,dest=path)
+
+By default, a local container image is created from the build result. The --output (or -o) option allows you to override this behavior and specify a custom exporter. For example, custom exporters allow you to export the build artifacts as files on the local filesystem instead of a container image, which can be useful for generating local binaries, code generation, etc.
+
+The value for --output is a CSV-formatted string defining the exporter type and options.
```

Comment:
CSV is a specific spreadsheet format. This is not quite that.

The dest= option is only documented with an example… maybe go all the way and say that it is a comma-separated sequence of key=value pairs, and make a list/table of the supported keys.

Reply:
fixed.